examples/vpc_service_control/onprem_project/variables.tf

```
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+variable "project_id" {}
```

Pleas add descriptions to all variables and run `terraform fmt` to format all TF files

---

Done.

---

Maybe the commit wasn't pushed, but this variable and all other variables still don't have a description see [here](https://github.com/mdgraboski/terraform-google-vpc-service-controls/blob/44e3f3c84ae4741d275af0e5b458528dd1fb6ac9/examples/vpc_service_control/onprem_project/variables.tf#L18)

---

I just realized why you had written "Done" on all changess regarding variables descriptions.
This suggestion shows you where the description of the variable should be.
This is part of our standards and it's also used to autogenerate documentation.
Once you've removed the comments and added the description in the description move on to this change to autogenerate the documentation.
Once you've added 
```suggestion
variable "project_id" {
 description =  "The ID of the GCP project that is going to be created"
}
```

---

Done.